src/Tizen.NUI/src/public/WindowSystem/QuickPanelClient.cs

```diff
@@ -0,0 +1,394 @@
+        private void OnVisibleChanged(int type, IntPtr ev_info, IntPtr data)
+        {
+            int _res;
+            IntPtr _state = IntPtr.Zero;
+            StateVisible state;
+
+            _res = Interop.QuickPanelClient.tzshQuickPanelEventVisibleGet(ev_info, ref _state);
+            _tzsh.ErrorCodeThrow(_res);
+
+            state = (StateVisible)_state;
+            _visibleChanged?.Invoke(this, state);
+        }
+
+        /// <exception cref="ArgumentException" > Thrown when failed of invalid argument.</exception>
+        private void OnOrientationChanged(int type, IntPtr ev_info, IntPtr data)
+        {
+            int _res;
+            IntPtr _state = IntPtr.Zero;
+            StateOrientation state;
+
+            _res = Interop.QuickPanelClient.tzshQuickPanelEventOrientationGet(ev_info, ref _state);
+            _tzsh.ErrorCodeThrow(_res);
+
+            state = (StateOrientation)_state;
+            _orientationChanged?.Invoke(this, state);
+        }
+
+        /// <summary>
+        /// Gets the visible state of the quickpanel.
+        /// </summary>
+        /// <returns>The visible state of the quickpanel service window.</returns>
+        /// <exception cref="ArgumentException" > Thrown when failed of invalid argument.</exception>
+        /// <exception cref="InvalidOperationException">Thrown when failed because of an invalid operation or no service.</exception>
+        public StateVisible GetVisible()
+        {
+            int _res;
+            IntPtr _vis = IntPtr.Zero;
+
+            _res = Interop.QuickPanelClient.tzshQuickPanelVisibleGet(_tzshQpClient, ref _vis);
+
+            _tzsh.ErrorCodeThrow(_res);
+
+            return (StateVisible)_vis;
+        }
+
+        /// <summary>
+        /// Gets the window's scrollable state of the quickpanel service window.
+        /// </summary>
+        /// <returns>The scrollable state of the quickpanel service window.</returns>
+        /// <exception cref="ArgumentException" > Thrown when failed of invalid argument.</exception>
+        /// <exception cref="InvalidOperationException">Thrown when failed because of an invalid operation or no service.</exception>
+        public StateScrollable GetScrollable()
+        {
+            int _res;
+            IntPtr _scroll = IntPtr.Zero;
+
+            _res = Interop.QuickPanelClient.tzshQuickPanelScrollableStateGet(_tzshQpClient, ref _scroll);
+
+            _tzsh.ErrorCodeThrow(_res);
+
+            return (StateScrollable)_scroll;
+        }
+
+        /// <summary>
+        /// Sets the window's scrollable state of the quickpanel service window.
+        /// </summary>
+        /// <exception cref="ArgumentException" > Thrown when failed of invalid argument.</exception>
+        /// <exception cref="InvalidOperationException">Thrown when failed because of an invalid operation or no service.</exception>
+        public void SetScrollable(StateScrollable scroll)
```

Comment:
Please add the necessary description to the method's parameter.

Reply:
I added param scroll description